mlflow/store/tracking/file_store.py

```diff
@@ -1610,24 +1614,28 @@ def delete_traces(
         deleted_traces = 0
         if max_timestamp_millis:
             trace_paths = list_all(traces_path, lambda x: os.path.isdir(x), full_path=True)
-            if max_traces is None:
-                max_traces = len(trace_paths)
+            trace_info_and_paths = []
             for trace_path in trace_paths:
                 try:
                     trace_info = self._get_trace_info_from_dir(trace_path)
                     if trace_info.timestamp_ms <= max_timestamp_millis:
-                        shutil.rmtree(trace_path)
-                        deleted_traces += 1
-                        max_traces -= 1
+                        trace_info_and_paths.append((trace_info, trace_path))
                 except MissingConfigException as e:
                     # trap malformed trace exception and log warning
                     request_id = os.path.basename(trace_path)
-                    logging.warning(
+                    _logger.warning(
```

Comment:
looks like we're using `logging.warning`. Can we fix this as a follow-up?

Reply:
or file a good-first issue :)

Reply:
Should use _logger everywhere in the file, will file a follow-up